core/src/main/java/io/questdb/griffin/model/QueryModel.java

```diff
@@ -306,7 +315,10 @@ public void clear() {
         clearSampleBy();
         orderBy.clear();
         orderByDirection.clear();
+        orderByAdvice.clear();
+        orderByDirectionAdvice.clear();
         orderByPosition = 0;
```

Comment:
Have you considered using OrderByMnemonic.ORDER_BY_UNKNOWN ?

Reply:
`orderByPosition` here stands for the position in the query text. Did you mean to use `OrderByMnemonic.ORDER_BY_UNKNOWN` in another line or I'm missing anything?

Reply:
I think code changed and line is not correct anymore . I meant 
```
orderByAdviceMnemonic = 0;
```

Reply:
My bad. Addressed in 5ad7205df202437b7c66622de11200ee569a4777